data/Place.csv

```diff
@@ -300,19 +313,19 @@ PL0357,Nanchang,28.682892,115.858197,,2018-07-10,WH,2018-07-10,WH
 PL0358,Sichuan Second Prison,29.58921,106.538559,The Sichuan Second Prison is now called Chongqing City Prison. PS00409,2018-08-15,WH,2018-09-10,DP
 PL0401,Skien,59.2086,9.60979,,2018-08-16,MG,2018-08-16,MG
 PL0402,Ann Arbor,42.280827,-83.743034,,2018-10-16,LH,2018-10-16,LH
-PL0403,Milano,45.463620,9.188120,,2018-10-19,MG,2018-10-19,MG
+PL0403,Milano,45.46362,9.18812,,2018-10-19,MG,2018-10-19,MG
 PL0404,Leipzig,51.339695,12.373075,,2018-10-26,MG,2018-10-26,MG
-PL0405,Zamość,50.717369,23.252760,,2018-10-26,MG,2018-10-26,MG
-PL0406,Laoting,22.887780,120.463560,,2018-10-26,MG,2018-10-26,MG
-PL0407,Pingchang,22.618220,120.566010,,2018-10-26,MG,2018-10-26,MG
-PL0408,Weddern,51.868540,7.307960,,2018-10-26,MG,2018-10-26,MG
-PL0409,Gallarta,43.320840,-3.072580,,2018-10-26,MG,2018-10-26,MG
-PL0410,Shuiyuan county,23.849670,110.400830,,2018-11-01,MG,2018-11-01,MG
-PL0411,Ostrogozhsk,50.860790,39.082700,,2018-11-01,MG,2018-11-01,MG
+PL0405,Zamość,50.717369,23.25276,,2018-10-26,MG,2018-10-26,MG
+PL0406,Laoting,22.88778,120.46356,,2018-10-26,MG,2018-10-26,MG
+PL0407,Pingchang,22.61822,120.56601,,2018-10-26,MG,2018-10-26,MG
+PL0408,Weddern,51.86854,7.30796,,2018-10-26,MG,2018-10-26,MG
+PL0409,Gallarta,43.32084,-3.07258,,2018-10-26,MG,2018-10-26,MG
+PL0410,Shuiyuan county,23.84967,110.40083,,2018-11-01,MG,2018-11-01,MG
+PL0411,Ostrogozhsk,50.86079,39.0827,,2018-11-01,MG,2018-11-01,MG
 PL0412,Voronezh,51.675495,39.208881,,2018-11-06,MG,2018-11-06,MG
-PL0413,Caprese,43.640960,11.985960,,2018-11-06,MG,2018-11-06,MG
-PL0414,Alloway,55.431090,-4.633100,,2018-11-06,MG,2018-11-06,MG
-PL0415,Hubei,37.598570,114.607580,,2018-11-09,MG,2018-11-09,MG
-PL0416,Warszawa ,52.229675,21.012230,,2018-11-09,MG,2018-11-09,MG
-PL0417,Pescasseroli,41.808250,13.789580,,2018-11-09,MG,2018-11-09,MG
-PL0999,unknown,0.00000,0.00000,,2018-11-13,MG,2018-11-13,MG
+PL0413,Caprese,43.64096,11.98596,,2018-11-06,MG,2018-11-06,MG
+PL0414,Alloway,55.43109,-4.6331,,2018-11-06,MG,2018-11-06,MG
+PL0415,Hubei,37.59857,114.60758,,2018-11-09,MG,2018-11-09,MG
+PL0416,Warszawa ,52.229675,21.01223,,2018-11-09,MG,2018-11-09,MG
+PL0417,Pescasseroli,41.80825,13.78958,,2018-11-09,MG,2018-11-09,MG
+PL0999,unknown,0,0,,2018-11-13,MG,2018-11-13,MG
```

Comment:
this shouldn't be a problem but i would disable auto-formatting here as well in LO